cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PageRankBenchmark.scala

```diff
@@ -24,4 +24,4 @@ class PageRankBenchmark(graph: DirectedGraph[Node],
   def operation() {
     PageRank(graph, pageRankParams)
   }
-}
\ No newline at end of file
+}
```

Comment:
Here are some benchmarks.  I ran each 1000 times.  Here are the benchmarks for the code that is currently in master:

<table>
  <tr>
    <td></tb><td><b>Personalized PR</b></td><td><b>Global PR</b></td>
  <tr>
    <td><b>Facebook Graph</b></tb><td>
0 ms 969 &mu;s</td><td>8 ms 52 &mu;s</td>
  </tr>
  <tr>
    <td><b>Wiki Graph</b></tb><td>1 ms 134 &mu;s</td><td>14 ms 778 &mu;s</td>
  </tr>
</table>

And affter adding the `linkanalysis` package.

<table>
  <tr>
    <td></tb><td><b>Personalized PR</b></td><td><b>Global PR</b></td><td><b>HITS</b></td>
  <tr>
    <td><b>Facebook Graph</b></tb><td>
1 ms 11 &mu;s</td><td>10 ms 542 &mu;s</td><td>107 ms 601 &mu;s</td>
  </tr>
  <tr>
    <td><b>Wiki Graph</b></tb><td>1 ms 247 &mu;s</td><td>18 ms 517 &mu;s</td><td>81 ms 611 &mu;s</td>
  </tr>
</table>